test/iterable_test.dart

```diff
@@ -164,6 +164,14 @@ void main() {
       expect(list1.contentEquals(list1), true);
     });
 
+    test('.containsAny()', () {
+      var list1 = const [1, 2, 3, 4, 5];
+      var list2 = const [1, 3, 5];
+
+      expect(list1.containsAll(list2), true);
+      expect(list2.containsAll(list1), false);
```

Comment:
doens't test `containsAny`

Reply:
Oops. Mixed up my functions and issues names

Reply:
Fixed